commit/merkleroot/rmn/client_test.go

```diff
@@ -48,21 +50,41 @@ func TestClient_ComputeReportSignatures(t *testing.T) {
 		resChan := make(chan RawRmnResponse, 200)
 		rawRmnClient := newMockRawRmnClient(resChan)
 
+		const numNodes = 4
+		rmnNodes := make([]RMNNodeInfo, numNodes)
+		for i := 0; i < numNodes; i++ {
+			publicKey, _, err := ed25519.GenerateKey(strings.NewReader(strings.Repeat("deterministic", 1000)))
```

Comment:
Haha, does it need to be repeated 1,000 times?

Reply:
That's definitely an interesting detail

Reply:
Should be way less, I'll find and use the minimum

Reply:
Fixed, added a 32char string